content/metadata.yaml

```diff
@@ -386,20 +383,19 @@ authors:
   - Supervision
   - Conceptualization
   affiliations:
-  - Division of Hematology, Oncology, Neuro-Oncology, and Stem Cell Transplant, Ann
-    & Robert H Lurie Children’s Hospital of Chicago
+  - Division of Hematology, Oncology, Neuro-Oncology, and Stem Cell Transplant, Ann & Robert H Lurie Children’s Hospital of Chicago
   - Department of Pediatrics, Northwestern University Feinberg School of Medicine
 - github: envest
   name: Steven M. Foltz
   initials: SMF
   email: steven.foltz@pennmedicine.upenn.edu
   orcid: 0000-0002-9526-8194
-  contributions: Validation
+  contributions:
+  - Validation
+  - Funding acquisition
```

Comment:
Because we're racing against the clock (cron job that alters the author metadata), I'm going to take this out for now and then we can discuss what the criteria for `Funding acquisition` are later today.

```suggestion
```